packages/ace-cli/src/index.js

```diff
@@ -45,6 +50,14 @@ function sleep(ms) {
 }
 
 async function run() {
+
+  if (cli.flags.rules) {
+    console.log('Write rules description to ./rules.html...');
+    fs.writeFileSync('./rules.html', util.getAllRulesAsHTML());
```

Comment:
I'm not sure we need rules to be written in HTML. A plain text list is fine, possibly a JSON format too.

If we want to keep the HTML option (in addition to the others), why not, but then I'd prefer to write the HTML to `system.out` rather than imposing the file name and path.

Or we can make `-R` accept an optional argument being the path of the output file. Maybe this gets complicated for just this option?